src/gradient_aware_harmonisation/convergence.py

```diff
@@ -153,8 +153,8 @@ def antiderivative(self) -> CosineDecaySplineHelperDerivative:
 @define
 class CosineDecaySplineHelperDerivative:
     """
-    Derivative of [CosineDecaySplineHelper][(m).]
-    """
+    Derivative of [CosineDecaySplineHelper][gradient_aware_harmonisation.convergence.CosineDecaySplineHelper]
```

Comment:
```suggestion
    Derivative of [CosineDecaySplineHelper][(p).CosineDecaySplineHelper]
```

Does it work if we change to this?  (docs suggest yes.. https://analog-garage.github.io/mkdocstrings-python-xref/1.16.2/)